app/controllers/bulk_updates_searches_controller.rb

```diff
@@ -18,7 +18,7 @@ def new
     collection_results = []
 
     if search_params.key?(:field) && search_params.key?(:query)
-      cmr_params[search_params[:field]] = search_params[:query].dup
+      cmr_params[search_params[:field].to_sym] = search_params[:query].dup
```

Comment:
So because this is using the value of the param, it was getting stored as a string, and when it was trying to match against the `wildcard_keys` in `hydrate_params`, they were not matching. This addition allowed the keys to match. Is there a better way this should be fixed?